doc/format.rst

```diff
@@ -549,6 +550,8 @@ not explicitly imported, `.. plot::` can be used directly if
 Documenting classes
 -------------------
 
+.. _classdoc:
```

Comment:
Is this a new / standard development?  I'm not familiar with it.

Reply:
I think it defines a label so I can refer to this section in the "Other points to keep in mind" section below. I'll have to check whether it rendered correctly.